src/lib/pay-request/typed_clients/services/admin_users/types.ts

```diff
@@ -14,6 +14,12 @@ export enum GoLiveStage {
   Live = 'LIVE'
 }
 
+export enum PSPTestAccountStage {
```

Comment:
Maybe useful to link to things like https://github.com/alphagov/pay-adminusers/blob/fe5a0223a7d047194616afa119cfd5a8a01655f0/src/main/java/uk/gov/pay/adminusers/model/PspTestAccountStage.java in the comments (while this isn't actually generated from the backends!)